front/components/vaults/VaultSideBarMenu.tsx

```diff
@@ -52,7 +52,7 @@ export default function VaultSideBarMenu({
   const groupedVaults = groupBy(vaults, (vault) => vault.kind);
   const sortedGroupedVaults = VAULTS_SORT_ORDER.map(
     (kind) => groupedVaults[kind] || []
-  );
+  ).filter((vaults) => vaults.length > 0);
```

Comment:
This will prevent displaying the "Create Vault" CTA. We might want to do this in the map block below.